src/systems/filecoin_mining/storage_mining/storage_miner_actor.go

```diff
@@ -1,21 +1,355 @@
 package storage_mining
 
-import base_mining "github.com/filecoin-project/specs/systems/filecoin_mining"
 import sector "github.com/filecoin-project/specs/systems/filecoin_mining/sector"
-import sealing "github.com/filecoin-project/specs/systems/filecoin_mining/sector"
+import block "github.com/filecoin-project/specs/systems/filecoin_blockchain/struct/block"
+import poster "github.com/filecoin-project/specs/systems/filecoin_mining/storage_proving/poster"
+
+// If a Post is missed (either due to faults being not declared on time or
+// because the miner run out of time, every sector is reported as failing
+// for the current proving period.
+func (sm *StorageMinerActor_I) CronAction() {
```

Comment:
`StorageMinerActor` is not called directly by `CronActor`. it is called by `StoragePowerActor`:

```
CronActor.EpochTick(vmr.Runtime)
|__ StoragePowerActor.EpochTick(vmr.Runtime)
    |__ StorageMnerActor.CheckPostSubmission()
```